_guides/analytics/analytics_basics.md

```diff
@@ -0,0 +1,229 @@
+In the following example,
+the page view request is sent to the URL,
+along with the current AMP document’s canonical URL, it’s title, and a
+[client ID](/docs/guides/analytics/analytics_basics.html#user-identification):
+
+{% highlight html linenos %}
+<amp-pixel src="https://example.com/analytics?url=${canonicalUrl}&title=${title}&clientId=${clientId(cid-scope)}"></amp-pixel>
+{% endhighlight %}
+
+Due to it’s simplicity,
+the `amp-pixel` tag can only include variables defined by the platform
+or that the AMP runtime can parse from the AMP page.
+The `amp-analytics` tag can include the same variables as `amp-pixel`,
+as well as uniquely defined variables inside the tag configuration. 
+
+Use the format `${varName}` in a request string for a page
+or platform-defined variable.
+The `amp-analytics` tag will replace the template with its actual value
+at the time of construction of the analytics request (see also
+[Variables supported in amp-analytics](https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/analytics-vars.md)).
+
+In the following example, the page view request is sent to the URL,
+with additional data extracted from variable substitutions,
+some provided by the platform,
+some within the `amp-analytics` configuration itself:
+
+{% highlight html linenos %}
+<amp-analytics>
+<script type="application/json">
+{
+  "requests": {
+    "pageview":"https://example.com/analytics?url=${canonicalUrl}&title=${title}&acct=${account}&clientId=${clientId(cid-scope)}",
+  },
+  "vars": {
+    "account": "ABC123",
+  },
+  "triggers": {
+    "some-event": {
+      "on": "visible",
+      "request": "pageview",
+      "vars": {
+        "title": "My homepage",
+        "clientId": "user12345"
+      }
+    }
+  }  
+}
+</script>
+</amp-analytics>
+{% endhighlight %}
+
+In the above example,
+the variables, `account`, `title`, and `clientID` are defined
+in the `amp-analytics` configuration.
+The `canonicalUrl` variable isn’t defined in the configuration,
+so it's value gets substituted by the platform.
+
+**Important:** Variable substitution is flexible;
+you can have the same variables defined in different locations,
+and the AMP runtime will parse the values in this order of precedence
+(see [Variable substitution ordering](/docs/guides/analytics/deep_dive_analytics.html#variable-substitution-ordering)).
+
+## User identification
+
+Websites use cookies to track a user’s browsing history;
+cookies tell the browser that a user visited a web page.
+In AMP,
+pages can be served from either a publisher's website or the AMP Cache.
+The publisher’s website and the AMP Cache have different domains.
+For security reasons,
+browsers can (and often will) limit access to another domain’s cookies
+(see also
+[Tracking users across origins](https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/cross-origin-tracking.md)).
+
+By default,
+AMP automatically creates a client ID to identify
```

Comment:
Edit starting from "By default"

"AMP will manage the provision of a client ID whether the page is accessed from the publisher's original website or through a cache".